hyp3_metadata/__main__.py

```diff
@@ -0,0 +1,207 @@
+import copy
+from datetime import datetime
+from pathlib import Path
+from typing import List, Tuple
+
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+from osgeo import gdal, osr
+
+from hyp3_metadata import __version__
+
+
+def get_environment() -> Environment:
+    env = Environment(
+        loader=PackageLoader('hyp3_metadata', '.'),
+        autoescape=select_autoescape(['html.j2', 'xml.j2']),
+        undefined=StrictUndefined,
+        trim_blocks=True,
+        lstrip_blocks=True,
+        keep_trailing_newline=True,
+    )
+    return env
+
+
+def render_template(template: str, payload: dict) -> str:
+    env = get_environment()
+    template = env.get_template(template)
+    rendered = template.render(payload)
+    return rendered
+
+
+def get_rtc_metadata_files(product_dir: Path, granule_name: str, dem_name: str, processing_date: datetime,
+                           looks: int, plugin_name: str, plugin_version: str, processor_name: str,
+                           processor_version: str) -> List[Path]:
+    payload = marshal_metadata(
+        product_dir=product_dir,
+        granule_name=granule_name,
+        dem_name=dem_name,
+        processing_date=processing_date.strptime('2020-01-01T00:00:00+0000', '%Y-%m-%dT%H:%M:%S%z'),
+        looks=looks,
+        plugin_name=plugin_name,
+        plugin_version=plugin_version,
+        processor_name=processor_name,
+        processor_version=processor_version,
+    )
+    files = []
+    files.extend(create_product_xmls(payload))
+    files.append(create_readme(payload))
+    files.append(create_dem_xml(payload))
+    files.append(create_browse_xml(payload))
+    files.append(create_inc_map_xml(payload))
+    files.append(create_ls_map_xml(payload))
+    return files
+
+
+def get_dem_resolution(dem_name: str) -> str:
+    data = {
+        'NED13': '1/3 arc seconds (about 10 meters)',
+        'NED1': '1 arc second (about 30 meters)',
+        'NED2': '2 arc seconds (about 60 meters)',
+        'SRTMGL1': '1 arc second (about 30 meters)',
+        'SRTMGL3': '3 arc seconds (about 90 meters)',
+    }
```

Comment:
need entries for `GIMP` and `REMA` here if we're supporting them in `get_dem_template_id` below